tests/sentry/integrations/slack/test_unfurl.py

```diff
@@ -436,10 +444,18 @@ def test_top_daily_events_renders_bar_chart(self, mock_generate_chart):
             data={"message": "second", "fingerprint": ["group2"], "timestamp": min_ago},
             project_id=self.project.id,
         )
-        debug_first_event = first_event.event_id  # noqa: F841
-        debug_second_event = second_event.event_id  # noqa: F841
-
-        url = f"https://sentry.io/organizations/{self.organization.slug}/discover/results/?field=message&field=event.type&field=count()&name=All+Events&query=message:[{first_event.message},{second_event.message}]&sort=-count&statsPeriod=24h&display=dailytop5&topEvents=2"
+        url = (
+            f"https://sentry.io/organizations/{self.organization.slug}/discover/results/"
```

Comment:
I still think 
```py
from urllib.parse import urlencode

base_url = f"https://sentry.io/organizations/{self.organization.slug}/discover/results/"
urlencode([
	("display", "dailytop5"),
	("field", "count()"),
	("field", "event.type"),
	("field", "message"),
	("name", "All+Events"),
	("query", f"message:[{first_event.message},{second_event.message}]"),
	("sort", "-count"),
	("statsPeriod", "24h"),
	("topEvents", "2"),
])
url = f"{base_url}?{params}"
```
might make this simpler to maintain.

Reply:
I actually tried this and got errors. Might've been syntax.